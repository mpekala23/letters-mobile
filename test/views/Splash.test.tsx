import * as React from "react";
import { SplashScreen } from "@views";
import { render, toJSON } from "@testing-library/react-native";

const AmeelioLogo = require("@assets/Ameelio_Logo.png");

describe("Splash screen", () => {
  it("should match snapshot", () => {
    const { container } = render(<SplashScreen />);
    const tree = toJSON(container);
    expect(tree).toMatchSnapshot();
  });

  it("should show the logo", () => {
    const { getByLabelText } = render(<SplashScreen />);
    expect(getByLabelText("Ameelio Logo")).toBeDefined();
  });
});
