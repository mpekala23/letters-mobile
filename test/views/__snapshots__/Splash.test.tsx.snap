// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash screen should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    accessible={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Image
      accessibilityLabel="Ameelio Logo"
      accessible={true}
      source={1}
      style={
        Object {
          "aspectRatio": 2.2768361581920904,
          "height": 120,
        }
      }
    />
  </View>
</View>
`;
